chromadb/test/quota/test_static_quota_enforcer.py

```diff
@@ -0,0 +1,78 @@
+import random
```

Comment:
Do you think it makes sense that limits-related tests are subject to the rigor of property tests?

Reply:
as there is no state, I think unit test is ok. wdyt?